### PR TITLE
implement delete by elasticsearch

### PR DESCRIPTION
**Pull request by tinnq1995**

### CODE REVIEW

File: index_test.go [hunk 1]
```diff
@@ -346,3 +346,56 @@ func TestUpdate(t *testing.T) {
...
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = client.Delete(context.TODO(), index, "1")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	insertedTweet, err := client.raw.Get().
```
Comment (TrungDM-HTV) [inline index_test.go]: insertedTweet -> deletedTweet

File: index.go [hunk 2]
```diff
@@ -111,6 +111,14 @@ func (i *IndexClient) DeleteIndex(ctx context.Context, index string) (*elastic.I
 	return i.raw.DeleteIndex(index).Do(ctx)
 }
 
+func (i *IndexClient) Delete(ctx context.Context, index string, id string) (*elastic.DeleteResponse, error) {
```
Comment (kazu1029) [inline index.go]: You could write `fucn (i *IndexClient) Delete(ctx context.Context, index, id string)` for short.

**Review by kazu1029 — CHANGES_REQUESTED** [review]
I had no problems with it in general, but I'm reviewing one point.

File: index.go (hunk 2 above)
Comment (TrungDM-HTV) [reply index.go]: Thank for your comment.
I fixed it,

**Review by kazu1029 — APPROVED** [review]
LGTM!

**Pull request merged**